snuba/admin/views.py

```diff
@@ -73,6 +75,16 @@ def health() -> Response:
     return Response("OK", 200)
 
 
+@application.route("/migrations/groups")
+def migrations_groups() -> Response:
+    res: List[Mapping[str, MigrationGroup | Sequence[str]]] = []
+    for migration_group in get_active_migration_groups():
+        if migration_group in settings.ADMIN_ALLOWED_MIGRATION_GROUPS:
+            group_migrations = get_group_loader(migration_group).get_migrations()
+            res.append({"group": migration_group, "migration_ids": group_migrations})
```

Comment:
I think you will want to return the state of each migration otherwise how would you show to the client which migrations they can run and which ones have already been applied ?

Reply:
I thought about that, I think that this will depend on the cluster they want to run the migration for. For now I'm going to leave this since there are checks in the runner that will prevent users from running migrations that have already been run and such, but yeah I think it would be ideal to show only the migrations that someone would be able to run successfully